manager/manager.go

```diff
@@ -475,9 +477,31 @@ func (m *manager) createContainer(containerName string) error {
 	}
 	glog.Infof("Added container: %q (aliases: %v, namespace: %q)", containerName, cont.info.Aliases, cont.info.Namespace)
 
+	contSpecs, err := cont.handler.GetSpec()
+	if err != nil {
+		return err
+	}
+
+	if contSpecs.CreationTime.After(m.startupTime) {
+		contRef, err := cont.handler.ContainerReference()
+		if err != nil {
+			return err
+		}
+
+		newEvent := &events.Event{
+			ContainerName: contRef.Name,
+			EventData:     cont,
```

Comment:
We probably don't want to return our internal container data structure here :) How about we return ContainerSpec instead since that is a user-facing structure.